pkg/cmd/roachtest/cluster.go

```diff
@@ -973,8 +977,9 @@ func (f *clusterFactory) newCluster(
 type attachOpt struct {
 	skipValidation bool
 	// Implies skipWipe.
-	skipStop bool
-	skipWipe bool
+	skipStop                   bool
+	skipWipe                   bool
+	tolerateRegistrationErrors bool
```

Comment:
`register` adds it to the cluster registry which we'd still want so our test runner code can keep going along no problem. We just don't want to stop/wipe the cluster in case of errors there.